frontend/packages/kubevirt-plugin/integration-tests/tests/vm.guest.agent.scenario.ts

```diff
@@ -70,7 +74,28 @@ describe('Tests involving guest agent', () => {
     execSync(`kubectl config set-context --current --namespace=default`);
   });
 
+  const environmentExpecScriptPath = `${KUBEVIRT_SCRIPTS_PATH}/guest-agent-login.sh`;
+  const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 3000 * 60 * 5;
+
   describe('Testing guest agent data', () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    it('create new vm and install guest agent, login and verify the loggedin users expose', async () => {
```

Comment:
How about updating the existed test to add check points for logged in user instead of writing a new test phase?